src/pages/About.jsx

```diff
@@ -117,16 +124,28 @@ function About() {
         ))}
       </Grid>
 
-      <Typography variant="h5" gutterBottom>
-        Looking for scholarships?
-      </Typography>
-      <Button
-        component={Link}
-        to="/scholarships"
-        variant="contained"
-        color="primary">
-        Browse Scholarships
-      </Button>
+      <Grid
```

Comment:
I'm guessing this component is similar to the one in UserHome.jsx right? We should probably refactor this into its own component file and reuse it across both pages.

Reply:
yes this is the part I mentioned in the other PR.